### DISCUSSION TITLE
Github profile not found my public url

### DISCUSSION THREAD

**Discussion by Muhammad-Awab** [community/community#68544] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I don't know my github profile not found when i search in google not found. I have 80+ repositories public and now not acessible. Help me please.
Here is my github profile link
https://github.com/Muhammad-Awab

**Comment by cehfisher** [comment]
Hi @Muhammad-Awab - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
